installing/installing_openstack/installing-openstack-installer-custom.adoc

```diff
@@ -30,6 +30,7 @@ include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
 include::modules/installation-initializing.adoc[leveloffset=+1]
 include::modules/installation-configuration-parameters.adoc[leveloffset=+1]
 include::modules/installation-osp-custom-subnet.adoc[leveloffset=+2]
+include::modules/installation-osp-deploying-bare-metal.adoc[leveloffset=+2]
```

Comment:
This module also applies to UPI.

Reply:
@mandre Just woke up, but does that mean that the "installer-provisioned network" description would need to go, then?

Reply:
@mandre And is the example YAML in it appropriate for both IPI and UPI?

Reply:
Actually, the flavors set in the `install-config.yaml` files aren't used in UPI. We need to modify the `os_flavor_master` and `os_flavor_worker` ansible variables in the `inventory.yaml` file in the UPI case.